src/routes/klageskjema/begrunnelse/begrunnelse.tsx

```diff
@@ -64,10 +68,23 @@ const Begrunnelse = ({ klage }: Props) => {
 
     useEffect(() => window.scrollTo(0, 0), []);
 
-    const [error, setError] = useState<string | null>(null);
-    const [attachmentsLoading, setAttachmentsLoading] = useState<boolean>(false);
-    const [attachmentError, setAttachmentError] = useState<string | null>(null);
-    const [submitted, setSubmitted] = useState<boolean>(false);
+    const createKlageUpdate = (klage: Klage): UpdateKlage => ({
+        id: klage.id,
+        tema: klage.tema,
+        ytelse: klage.ytelse,
+        saksnummer: klage.saksnummer,
+        fritekst,
+        vedtak: dateToVedtakText(chosenDateOption, chosenISODate)
+    });
+
+    useEffect(() => {
+        const timeout = setTimeout(async () => {
```

Comment:
Hm, nå ser jeg at det er mulig å forkorte den enda mer:
`const timeout = setTimeout(
            () => updateKlage(createKlageUpdate(klage)).catch((error: CustomError) => setError(error.message)),
            1000
        );`

Hva tenker du, er dette bedre eller bare kortere?

Reply:
`await` gjør vel egentlig ingenting i denne funksjonen, om `await` fjernes kan `async` også fjernes.

Reply:
> Hm, nå ser jeg at det er mulig å forkorte den enda mer:
> `const timeout = setTimeout( () => updateKlage(createKlageUpdate(klage)).catch((error: CustomError) => setError(error.message)), 1000 );`
> 
> Hva tenker du, er dette bedre eller bare kortere?

Ha ha :) helt greit for meg.

Reply:
Var virkelig ment for diskusjon :)
Det er alltid en tradeoff med oneliners. Lesbarheten går ofte raskt nedover.